Bottlenose/bottlenose.cpp

```diff
@@ -0,0 +1,132 @@
+#include <bottlenose.h>
+//DEFAULTS
+int _currentPin = 8;
+int _msDot = 50;
+int _msDash = 350;
+int _msCharGap = 100;
+int _highDelay = 2;
+int _lowDelay = 3;
+
+int _digits[10][5] ={
+    {1,1,1,1,1}, // 0
+    {0,1,1,1,1}, // 1
+    {0,0,1,1,1}, // 2
+    {0,0,0,1,1}, // 3
+    {0,0,0,0,1}, // 4
+    {0,0,0,0,0}, // 5
+    {1,0,0,0,0}, // 6
+    {1,1,0,0,0}, // 7
+    {1,1,1,0,0}, // 8
+    {1,1,1,1,0}, // 9
+};
+//CONSTRUCTOR
+bottlenose::bottlenose(void)
+{
+  //ADD STUFF
+}
+//PUBLIC
+bool bottlenose::begin(int pin)
+{
+  _currentPin = pin;
```

Comment:
I'd say we should either put this in the constructor, or create a singleton object to be used (like Serial or SD) instead of making the client instantiate the class.